tests/calc/files/parameters/case_parameters.py

```diff
@@ -62,6 +63,16 @@ def _get_poscar_info(name):
            'BULK_LIKE_BELOW': 0.35, 'T_DEBYE': 420, 'T_EXPERIMENT': 100,
            'SITE_DEF': {'Ir': {'surf': {3, 2}}, 'O': {'ads': {1}}},
            'VIBR_AMP_SCALE': ['*surf 1.3',], 'V0_IMAG': 5.0},
+    'PtRh': {'RUN': [0],
```

Comment:
```suggestion
    'PtRh': {
        'RUN': [0],
```

Sorry, one more I just noticed.